packages/sol-apy-sdk/index.ts

```diff
@@ -100,5 +106,10 @@ export const calcAverageApy = (
     return null;
   }
 
-  return priceChange ** (millisecondsInAYear / deltaMilliseconds) - 1;
+  return {
+    apy: priceChange ** (millisecondsInAYear / deltaMilliseconds) - 1,
+    epochs: priceRecords.length < epochs ? priceRecords.length : epochs,
+    startTimestamp: timestampStart,
```

Comment:
please refactor so either timestamp* or *timestamp is used in all places to avoid confusion in the future